python/pacemaker/_cts/patterns.py

```diff
@@ -201,7 +201,8 @@ def __init__(self):
             r"Faking parameter digest creation",
             r"Parameters to .* action changed:",
             r"Parameters to .* changed",
-            r"pacemakerd.*\[[0-9]+\] terminated( with signal| as IPC server|$)",
+            r"pacemakerd.*\[[0-9]+\] terminated( with signal|$)",
```

Comment:
It would really be nice to have a better way for `cts-lab` to track events than looking for specific log strings... not saying that would be easy or even feasible.